pages.de/common/minisign.md

```diff
@@ -0,0 +1,24 @@
+# minisign
+
+> Ein denkbar einfaches Werkzeug um Dateien zu signieren und Signaturen zu verifizieren.
+> Mehr Informationen: <https://jedisct1.github.io/minisign/>.
+
+- Ein neues Schlüsselpaar im Standardpfad generieren:
+
+`minisign -G`
+
+- Eine Datei signieren:
+
+`minisign -Sm {{/pfad/zu/datei}}`
+
+- Eine Datei signieren und dabei einen vertrauenswürdigen (signierten) und einen nicht vertrauenswürdigen (unsignierten) Kommentar in der Signatur anfügen:
```

Comment:
There's two of the same word (`vertrauenswürdigen`) here—is that intentional?  The English line contains `trusted` and `untrusted` 🤔

Reply:
Yes, this is intended. "Vertrauenswürdig" means "trusted" (or actually "trustworthy", but the literal translation for "trusted" would be "vertraut", which doesn't really make sense in German in this context), "nicht" means "not", thus, the latter is "not trusted". I *could* translate it literally as "unvertrauenswürdig", however, I think this would be considered somewhat poor style. Thanks for pointing this out anyway, though.

Reply:
Ah, I see! Thank you for this in-depth explanation! ❤️